scripts/install

```diff
@@ -65,7 +64,7 @@ ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 #=================================================
 ynh_script_progression --message="Storing installation settings..."  --weight=1
 
-ynh_app_setting_set --app=$app --key=domain --value=$domain
+ynh_app_setting_set --app=$app --key=domain --value=none.ynh.fr
```

Comment:
Uuuuuh I really doubt it's a good thing to define domain with a dummy value ... It's very likely to make plenty of things inside yunohost that will expect that this domain is actually registered somewhere...